client/state/sites/hooks/use-site-global-styles-status.ts

```diff
@@ -19,42 +59,28 @@ const DEFAULT_GLOBAL_STYLES_INFO: GlobalStylesStatus = {
 	globalStylesInPersonalPlan: false,
 };
 
-function isObject( x: unknown ): x is Record< string, unknown > {
-	return typeof x === 'object' && x !== null;
-}
-
-const getExperimentAssignment = ( experimentName: string ): string | null => {
-	return wpcom.req
-		.get(
-			{
-				path: '/experiments/0.1.0/assignments/calypso',
-				apiNamespace: 'wpcom/v2',
-			},
-			{
-				experiment_name: experimentName,
-			}
-		)
-		.then( ( response: unknown ) =>
-			isObject( response ) &&
-			isObject( response.variations ) &&
-			typeof response.ttl === 'number' &&
-			0 < response.ttl
-				? response.variations[ experimentName ]
-				: null
-		)
-		.catch( () => null );
-};
+const getGlobalStylesInfoForSite = ( siteId: number | null ): Promise< GlobalStylesStatus > => {
+	if ( ! shouldRunGlobalStylesOnPersonalExperiment( siteId ) ) {
+		return new Promise( ( resolve ) =>
+			resolve( {
+				shouldLimitGlobalStyles: true,
+				globalStylesInUse: false,
+				globalStylesInPersonalPlan: false,
+			} )
+		);
```

Comment:
Not a blocker. Here maybe we could replace `new Promise...` with `return Promise.resolve...`